src/plots/layout_attributes.js

```diff
@@ -188,6 +188,29 @@ module.exports = {
         editType: 'calc',
         description: 'Sets the default trace colors.'
     },
+    colorscale: {
+        sequential: {
+            valType: 'colorscale',
+            dflt: false,
```

Comment:
What would happen if we set `dflt: scales.RdBu` ?

Reply:
That's another way of doing it and I think it's much better than the current approach. We'd have the defaults clearly defined in the attributes where it should be :tada: